connector_cli/oauth2login.py

```diff
@@ -49,65 +50,44 @@ def login_callback():
     }
     resp = requests.post(token_url, data=the_data)
     data = resp.json()
-
     secrets = {
         "oauth_access_token": data["access_token"],
         "oauth_refresh_token": data["refresh_token"],
         "oauth_client_id": client_id,
         "oauth_client_secret": client_secret,
     }
-
-    is_failed_params = False
-    # post secrets
-    for secret, value in secrets.items():
-        response = requests.post(service_url + "/systems/%s/secrets" % system_placeholder,
-                                 headers={"Authorization": "Bearer %s" % service_jwt}, json={secret: value})
-        if response.status_code == 200:
-            print("Updated secret: %s successfully" % secret)
-        else:
-            is_failed_params = True
-            print("Failed to update secret: %s" % secret)
-            print(response.text)
-
-    # update env
-
-    env = requests.get(service_url + "/env", headers={"Authorization": "Bearer %s" % service_jwt}).json()
+    # put secrets
+    system = sesam_node.api_connection.get_system(system_id)
+    system.put_secrets(secrets)
```

Comment:
When using the sesamclient module, the general pattern is to wrap these calls (or some outer scope) in a try: except: clause to give feedback to the user (logging, prints or similar) just to indicate when something went wrong - in the old code in the diff there seems there was some prints() to that effect, but perhaps we should reuse the logger we set up in the main sesam.py module and use the logging facilities for this here?